script.py

```diff
@@ -1,9 +1,45 @@
 from pdd.model import PDDModel
 import torch
 
-def get_predict():
+from sklearn.neighbors import KNeighborsClassifier
+from sklearn.metrics import accuracy_score
+
+from PIL import Image
+import numpy as np
+from skimage.transform import resize
+import torch
+from scipy.spatial.distance import cosine
+import pickle
+
+def load_image(infilename) :
+    img = Image.open( infilename )
+    img.load()
+    data = np.asarray( img, dtype="int32" )
+    return data
+
+def get_predict(img_name):
+
+    device = torch.device('cpu')
+    # device = 'cuda' if torch.cuda.is_available() else 'cpu'
     model=PDDModel(1280,15,True)
-    model.load_state_dict(torch.load(PATH))
+    knn=KNeighborsClassifier(3,metric=cosine)
```

Comment:
Maybe this is not necessary because you are loading the whole model in the line 35.